metadata-ingestion/docs/sources/delta-lake/delta-lake.md

```diff
@@ -119,7 +119,10 @@ Create a datahub ingestion yaml file (delta.s3.dhub.yaml) to ingest metadata fro
 source:
   type: "delta-lake"
   config:
-    base_path:  "s3://my-bucket/my-folder/sales-table"
+    base_path:  "s3://my-bucket/my-folder/sales-table" 
+    or
```

Comment:
Added an additional example demonstrating the usage of path_spec.

Reply:
Sagar when the change is rendered it is going to be very hard to see the html. Please take your English comments outside the recipe instead of within the recipe itself and add another recipe below for the new case.

Reply:
Added now it will look like below

<img width="1434" alt="Screenshot 2024-06-14 at 5 49 35 PM" src="https://github.com/datahub-project/datahub/assets/159135491/e6d69e76-55b2-4e86-b72d-cea9cc220f0b">